plugins/usd/module/vtkF3DUSDImporter.cxx

```diff
@@ -192,25 +195,34 @@ class vtkF3DUSDImporter::vtkInternals
           polydata = this->MeshMap[meshPrim.GetPath().GetAsString()];
           bool meshAlreadyExists = (polydata != nullptr);
 
-          if (!meshAlreadyExists)
-          {
-            polydata = vtkSmartPointer<vtkPolyData>::New();
-          }
-
-          // read normals
+          // attributes
           pxr::UsdAttribute normalsAttr = meshPrim.GetNormalsAttr();
+          pxr::UsdAttribute pointsAttr = meshPrim.GetPointsAttr();
+          pxr::UsdAttribute facesCountAttr = meshPrim.GetFaceVertexCountsAttr();
+          pxr::UsdAttribute facesIndicesAttr = meshPrim.GetFaceVertexIndicesAttr();
+
+          std::vector<pxr::UsdGeomPrimvar> primVars =
+            pxr::UsdGeomPrimvarsAPI(meshPrim).GetPrimvars();
+
+          auto TimeVarying = [](const auto& a) { return a.ValueMightBeTimeVarying(); };
```

Comment:
i dont see the point of thisn lambda, why not use the expression directly ?

Reply:
For `std::any_of` usage